multipaz/src/commonTest/kotlin/org/multipaz/mdoc/engagement/EngagementGeneratorTest.kt

```diff
@@ -46,33 +45,6 @@ class EngagementGeneratorTest {
         assertEquals(0, engagement.originInfos.size.toLong())
     }
 
-    @Test
-    @Throws(Exception::class)
-    fun testWebsiteEngagement() {
```

Comment:
Why's this being removed?

Reply:
Because we're moved MdocConnectionMethodHttp to multipaz-android-legacy.